daisy/workflow/step_attach_disks.go

```diff
@@ -24,7 +24,14 @@ type AttachDisk struct {
 	Disk, Instance string
 }
 
+func (a *AttachDisks) populate(s *Step) error {
+	return nil
+}
+
 func (a *AttachDisks) validate(s *Step) error {
+	if err := a.populate(s); err != nil {
```

Comment:
Why is this in step.validate? step.populate should be called from w.populateStep (theres code there that probably should be replaced with this anyway)